packages/react/src/components/Table/Table.jsx

```diff
@@ -426,6 +426,7 @@ export const defaultProps = (baseProps) => ({
     itemSelected: (selectedCount) => `${selectedCount} item selected`,
     rowCountInHeader: (totalRowCount) => `Results: ${totalRowCount}`,
     toggleAggregations: 'Toggle aggregations',
+    toolbarLabelAria: '',
```

Comment:
Why default to empty string? Do you need a default at all here?

Reply:
Following the established patterned, See L432 (3 lines below)

Reply:
I see, good that you noticed the default for `emptyMessageBody` but that is not a pattern we want to use here (we should change `emptyMessageBody` to use `undefined`). The best option, if you are going with the breaking change, is to use `undefined` as default value here as well.

Reply:
I updated `toolbarLabelAria` to default to `undefined`. I also changed the commit message to indicate it is a breaking change. Depending on how breaking changes are handled in this repo / how often major release are released, I may create an additional PR that doesn't change the default (just to get that out more quickly. I still believe this change to the default is better and should be used.)